testing/howard-bc/APKBUILD

```diff
@@ -32,11 +32,13 @@ doc() {
 	# Doc files
 	install -Dm644 "$builddir"/LICENSE.md \
 			"$subpkgdir"/usr/share/doc/$pkgname/LICENSE.md
+
+	rm -rf "$pkgdir"/usr/share/man/
 }
 
 package() {
 	cd "$builddir"
 	make install
```

Comment:
```suggestion
	make install
	
	install -Dm644 LICENSE.md \
		"$pkgdir"/usr/share/doc/$pkgname/LICENSE.md
```

Reply:
Should be resolved by the recent changes.